packages/utils/typescript/tsconfigs/server.json

```diff
@@ -11,6 +11,7 @@
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
 
+    "tsBuildInfoFile": "./.tsbuildinfo",
```

Comment:
Shouldn't we add this to the generated project gitignore files?

Reply:
Yes, good catch!

Reply:
I'll make another PR for that